### PR TITLE
APHL key server integration

### PR DESCRIPTION
**Pull request by colmharte**
Adds support for downloading exposure files from APHL server in US which is based on google reference server for key server.

Code now supports nearform key server and google key server.

### CODE REVIEW

File: android/src/main/java/ie/gov/tracing/nearby/StateUpdatedWorker.java [hunk 1]
```diff
@@ -121,8 +121,15 @@ public StateUpdatedWorker(
 
               int[] ad = exposureSummary.getAttenuationDurationsInMinutes();
 
-              double[] tw = doubleArrayFromString(SharedPrefs.getString("thresholdWeightings", context));
-              long timeThreshold = SharedPrefs.getLong("timeThreshold", context);
+              double[] tw;
+              long timeThreshold;
+              if (simulate) {
```
Comment (floridemai) [inline android/src/main/java/ie/gov/tracing/nearby/StateUpdatedWorker.java:126]: This is a missing piece from the previous PR?

**Review by floridemai — APPROVED** [review]
Wasn't able to test it but the  code looks good if the google file processing logic is the correct one.

File: README.md [hunk 2]
```diff
@@ -124,6 +124,8 @@ Use to configure the module. This method is synchronous, and should be called be
 
 - `exposureCheckFrequency`: a number representing the period between exposure downloads in minutes
 - `serverURL`: a string representing the the server api url (should not have trailing /)
+- `keyServerUrl`: a string representing the the key server api url (should not have trailing /). Will defauilt to serverURL
```
Comment (fiacc) [inline README.md]: typo - default

**Review by fiacc — APPROVED** [review]
LGTM

File: android/src/main/java/ie/gov/tracing/nearby/StateUpdatedWorker.java (hunk 1 above)
Comment (colmharte) [reply android/src/main/java/ie/gov/tracing/nearby/StateUpdatedWorker.java:126]: Yes found a bug in simulate where it failed on Android if you have never loaded a config file

File: README.md (hunk 2 above)
Comment (colmharte) [reply README.md]: fixed

**Pull request merged**